js/authentication.md

```diff
@@ -1075,9 +1075,9 @@ const federated = {
 
 There is also `withGoogle`, `withFacebook`, `withAmazon` components, in case you need to customize a single provider.
 
-### Using Amazon Cognito Hosted UI
+### Federated with Cognito User Pool Service
```

Comment:
Can you update to "Federate with Cognito User Pools"

Reply:
Done